applications/serial_lte_modem/prj.conf

```diff
@@ -72,6 +72,7 @@ CONFIG_SETTINGS_FCB=y
 CONFIG_FCB=y
 
 # nRF Cloud
+#CONFIG_NRF_CLOUD_IPV6=y
```

Comment:
In the commit message, it says "default not defined". However, here it is defined.

Reply:
It's commented out by default.
Customer could uncomment out this line for quick test of nRF Cloud in IPv6 network.